packages/ChangesReloaded-Core.package/CRChangeListItem.class/instance/doesNotUnderstand..st

```diff
@@ -1,4 +1,4 @@
-accessing
+error handling
```

Comment:
This doesn't have to do something with error handling, but is the Squeak way of wrapping a class: We can handle a CRChangeListItem exactly like a ChangeRecord, without the need to define all methods of a ChangeRecord on the CRChangeListItem (i.e. calling CRChangeListItem isInImage). All messages that are not understood by the CRChangeListItem are redirected to self changeRecord, the encapsulated ChangeRecord

So it's actually part of accessing, in this case

Reply:
Wouldn't call an expected behavior error tho